building/devices/nuvoton.rst

```diff
@@ -0,0 +1,61 @@
+.. _nuvoton:
+
+#######
+NUVOTON
+#######
+
+The instructions here will tell how to build and run OP-TEE for Nuvoton platform
+standalone (not as a part of openbmc image).
+
+
+.. _nuvoton_build_instructions:
+
+Build instructions
+******************
+
+=================
+Pre-requirements:
+=================
+
+#. Download and install the latest IGPS from the Nuvoton Israel github, i.e.,
+   $ git clone https://github.com/Nuvoton-Israel/igps-npcm8xx
+
+#. Download and extract the ARM GNU Toolchain 11.3 for aarch64
+   $ wget https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-aarch64-none-elf.tar.xz?rev=73ff9780c12348b1b6772a1f54ab4bb3&hash=B80470312E67CF1F157D0A883FDEB279
+   $ tar xvf arm-gnu-toolchain-11.3.rel1-x86_64-aarch64-none-linux-gnu.tar.xz
+
+#. Add the ARM GNU Toolchain binary to the $PATH, for example:
+   $ vi .cshrc
+   set path = ( . ~/bin /home/bin /usr/local/bin /bin /usr/bin /work/cross-compiler/arm-gnu-toolchain-11.3.rel1-x86_64-aarch64-none-elf/bin)
```

Comment:
I imagine most Linux users use bash, not csh, but it doesn't matter much, I suppose anyone can figure out what to do.

Reply:
Fully agree, fixed